plugins/woocommerce/tests/legacy/unit-tests/rest-api/Tests/Version2/system-status.php

```diff
@@ -21,14 +21,26 @@ class WC_Tests_REST_System_Status_V2 extends WC_REST_Unit_Test_Case {
 	 */
 	public function setUp(): void {
 		parent::setUp();
-		$this->endpoint = new WC_REST_System_Status_Controller();
-		$this->user     = $this->factory->user->create(
+		$this->user = $this->factory->user->create(
 			array(
 				'role' => 'administrator',
 			)
 		);
 	}
 
+	/**
+	 * Fetches the system status data and caches it.
+	 */
+	private function get_system_status_data() {
```

Comment:
I'm a little worried that this method name might not adequately communicate what it does. `get_system_status_data()` feels like of ambiguous, maybe something that more fully describes the context we're using this in?

I don't have a great suggestion unfortunately! Since there's a test that gets the status for user 0, maybe `get_system_status_for_user( $id )` and pass `$this->user->id`? It would require changing the variable to support multiple users, but, it might be more descriptive.